code/modules/paperwork/paper.dm

```diff
@@ -11,6 +11,8 @@
  **/
 /obj/item/paper
 	var/const/MAX_PAPER_LENGTH = 1000
+	var/const/MAX_PAPER_STAMPS = 30		// Too low?
+	var/const/MAX_PAPER_STAMPS_OVERLAYS = 4
```

Comment:
Please don't use `const` as it creates runtime overhead. Use #defines instead

Reply:
Ah ok, I will switch them